@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFLTDraw.sqf

```diff
@@ -24,6 +24,9 @@ private _airspeed = vectorMagnitude (velocity _heli vectorDiff wind);
 private _baroAlt  = getPosASL _heli # 2 * SCALE_METERS_FEET;
 private _radarAlt = getPos _heli # 2 * SCALE_METERS_FEET;
 _heli setUserMFDText [MFD_INDEX_OFFSET(MFD_TEXT_IND_FLT_BALT), (round (_baroAlt / 10) * 10) toFixed 0];
+if (_radarAlt > 50) then {
```

Comment:
If the logic for formatting the radar altitude is identical across the MPD FLT Page and IHADSS Symbology, could we share it?

Reply:
Fixed